src/main/java/com/google/firebase/remoteconfig/RemoteConfigUtil.java

```diff
@@ -20,20 +20,59 @@
 
 import com.google.common.base.Strings;
 
+import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.Date;
 import java.util.TimeZone;
 
 final class RemoteConfigUtil {
 
+  private static final String ZULU_DATE_PATTERN = "yyyy-MM-dd'T'HH:mm:ss.SSS000000'Z'";
```

Comment:
Add a comment here indicating why we need 2 zulu format strings.

Reply:
Good call. Thanks!